source/external-interface-design/external-interface-design.rst

```diff
@@ -155,21 +154,16 @@ This includes:
 
 -  IoT devices
 
--  Service profiles
-
 -  Device profiles
 
+-  Multicast groups.
+
 Changes to these entities must always happen in OS2iot, which is then
 synchronized to Chirpstack. It is not supported to change data directly
 in Chirpstack. All manipulation of settings in the Chirpstack is thus
-done via the Chirpstack API. If locally running on port 8080 then Chirpstack Application Services Swagger UI is available at http://localhost:8080/api
+done via the Chirpstack API. The protocol documentation for the api is documented at https://www.chirpstack.io/docs/chirpstack/api/api.html.
 
-There is no direct mapping between applications in OS2iot and applications in Chirpstack, even though they cover a similar concept. 
-Instead, a separate Chirpstack application is created for each separate ServiceProfile created through OS2iot. 
-
-The new Chirpstack application is created the first time a device with a previously unused ServiceProfile is added to an OS2iot application.
-This is illustrated in the flowchart below:
-|image13|
+After the update to Chirpstack V4, applications on OS2IoT are also created one-to-one on Chirpstack.
```

Comment:
Delete this sentence. (Or probably better: have in in some migration guide section)

Reply:
Might be a good idea to make a seperate section to migration :-)?